tf2/include/tf2/buffer_core.h

```diff
@@ -92,8 +92,9 @@ class BufferCore
 {
 public:
   /************* Constants ***********************/
+  static const int DEFAULT_CACHE_TIME = 10;  //!< The default amount of time to cache data in seconds
```

Comment:
we're using https://github.com/ros2/geometry2/blob/c5b4d4738ec1f824af364c9e55fb35239de3f1c6/tf2/include/tf2/buffer_core.h#L71 instead of this